checker/src/org/checkerframework/checker/oigj/OwnershipAnnotatedTypeFactory.java

```diff
@@ -3,6 +3,7 @@
 import org.checkerframework.checker.oigj.qual.Dominator;
 import org.checkerframework.checker.oigj.qual.Modifier;
 import org.checkerframework.checker.oigj.qual.O;
+import org.checkerframework.checker.oigj.qual.OIGJMutabilityBottom;
```

Comment:
This one is actually need because I moved OIGJMutabilityBottom to the qual package.